gotham/src/helpers/http/response/mod.rs

```diff
@@ -1,14 +1,15 @@
 //! Helpers for HTTP response generation
 
-use hyper::header::{ContentLength, ContentType, Location};
-use hyper::{Method, Response, StatusCode};
+use http::response;
+use hyper::header::{HeaderMap, HeaderName, CONTENT_LENGTH, CONTENT_TYPE, LOCATION,
+                    X_CONTENT_TYPE_OPTIONS, X_FRAME_OPTIONS, X_XSS_PROTECTION};
+use hyper::{Body, Method, Response, StatusCode};
 use mime::Mime;
 use std::borrow::Cow;
 
-use helpers::http::header::{XContentTypeOptions, XFrameOptions, XRequestId, XXssProtection};
 use state::{request_id, FromState, State};
 
-type Body = (Vec<u8>, Mime);
+type DataMime = (Vec<u8>, Mime);
```

Comment:
This is a better name, but I question whether we really need this type. It's only used a couple of times?